test/x/track_test.rb

```diff
@@ -4,11 +4,8 @@
 module X
   class TrackTest < Minitest::Test
     def test_tracks
-      f= './test/fixtures/xapi_v3_tracks.json'
```

Comment:
If you run the x-api repository locally and fire it up with `RACK_ENV=test bundle exec rackup -p9292` then you should be able to grab the fixture data instead of the production data for your fixture. I don't know if that creates other problems, though.